src/TreeHouse/BaseApiBundle/Behat/ApiContext.php

```diff
@@ -321,12 +321,21 @@ public function theResultKeyShouldEqualNode($key, $node)
     }
 
     /**
-     * @Then the api error should contain ":text"
+     * @Then the api/API error should contain :text
```

Comment:
Why not make this a bit more strict (one or the other) so we have consistent steps?